app/jobs/change_pickup_library_job.rb

```diff
@@ -5,7 +5,7 @@ class ChangePickupLibraryJob < ApplicationJob
 
   def perform(hold_key:, session_token:, pickup_library:)
     symphony_client = SymphonyClient.new
-    redis_client = Redis.new
+
```

Comment:
note to self: remember to make this change for renewals after merging this PR  :)